src/components/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,26 @@
+import React, {useState} from 'react';
+import classes from './Checkbox.module.scss';
+
+const Checkbox = (props) => {
+    const [checkbox, setCheckbox] = useState(false);
```

Comment:
const [isChecked, setIsChecked] = ...

it is better to use "is...", "has..." for such props and state naming

Reply:
done